addons/coupon/models/coupon.py

```diff
@@ -95,6 +95,10 @@ def cron_expire_coupon(self):
         expired_ids = [res[0] for res in self._cr.fetchall()]
         self.browse(expired_ids).write({'state': 'expired'})
 
+    def _get_partner_to_default_template(self):
+        default_template = self._get_default_template()
+        return default_template._render_template(default_template.partner_to, 'coupon.coupon', self.ids)[self.id]
```

Comment:
@Williambraecky i think i'm just taking the template's partner_to field, would it be better if i edited like this
```suggestion
        return default_template._render_template(default_template.partner_to, 'coupon.coupon', [self.id,])[self.id]
```

Reply:
It's still the wrong way to go about this issue.
Follow my first review for a proper fix.